src/utils/logGraphQLRequests.ts

```diff
@@ -6,12 +6,13 @@ const logGraphQLRequests = {
     const noNewlinesQuery: string = requestContext.request.query
       .replace(/\n/g, '')
       .replace(/\s{2,}/g, ' ')
-    logger.debug(
-      'Request Query:\n' +
-        noNewlinesQuery +
-        '\n Request Variables: \n ' +
-        JSON.stringify(requestContext.request.variables)
-    )
+
+    // Redacted copy of the variables for logging purposes
+    const variables = { ...requestContext.request.variables }
+    if (variables?.orgToken) {
+      variables.orgToken = '[REDACTED]'
+    }
```

Comment:
Are we using this variable anywhere in the codebase? If not why don't we remove it?